src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,66 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class LoginPage extends BasePage {
+
+    By emailField = By.cssSelector("[type='email']");
+    static By passwordField = By.cssSelector("[type='password']");
+    By submitButton = By.cssSelector("button[type='submit']");
+    By successBanner = By.cssSelector(".success");
+    static By profile = By.cssSelector(".view-profile");
+    By userName = By.cssSelector(".view-profile .name");
+    By nameInput = By.cssSelector("#inputProfileName");
+    By passwordInput = By.cssSelector("#inputProfileCurrentPassword");
+    By saveButton = By.cssSelector(".btn-submit");
```

Comment:
I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```